README.md

```diff
@@ -167,9 +164,9 @@ For more information, see [Use Client-Side SSL Certificates for Authentication b
 
 ## 5. Test the API Gateway Integration
 
-The next step is to verify that the API in API Gateway is integrated with your application. The [Postman](https://www.getpostman.com/) app is a convenient testing tool for this because it provides fields for adding your signing information to the HTTPS request.
+The next step is to verify that the API in API Gateway is integrated with your application. The [Postman](https://www.getpostman.com/) app is a convenient testing tool for verifying this because it provides fields for adding your signing information to an HTTPS request.
```

Comment:
1. should we say -

provide fields for adding your [Signature Version 4](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html) signing information to an HTTPS request


Just calling out that they need to work with SigV4 specifically and Postman makes it easy.

2. minor: 'testing' and 'verifying' seem little redundant